docker/standalone.php

```diff
@@ -52,6 +52,14 @@ function classifySpeed(value) {
                 if (value >= 10) return 2;
                 return 1;
             
+        }
+
+		function classifySpeedUl(value) {
```

Comment:
make it `classifySpeedUpload` avoid short forms

Reply:
changes are made, kindly validate